styles/pages/map-selector/map-entry.scss

```diff
@@ -226,4 +226,24 @@ $transition: 0.125s ease-out 0s;
 			@include mixin.gradient-themed($color: 'purple', $direction: 'down');
 		}
 	}
+
+	&__lobby {
+		width: 100px;
+		height: 20px;
+		flow-children: left;
+		align: right bottom;
+
+		&-icon {
+			height: 100%;
+			width: height-percentage(100%);
+			img-shadow: $button-icon-shadow;
+		}
+
+		&-playercount {
+			font-size: 14px;
```

Comment:
From screenshot you posted in Discord, I think this look a bit better with `font-size: 16px` and `font-weight: medium` - text is looks a bit small